website/public/phpinfo.php

```diff
@@ -0,0 +1,3 @@
+<?php 
+
+phpinfo();
```

Comment:
Must fix: This is a security issue. Delete.

We shouldn't even be serving PHP on https://quick-lint-js.com/. (But that's not your patch's fault.)

Reply:
done